package.json

```diff
@@ -24,6 +25,7 @@
     "chai": "3.3.0",
     "grunt": "0.4.5",
     "grunt-eslint": "17.1.0",
+    "grunt-jscs": "^2.1.0",
```

Comment:
Something we did in payments was to turn off the '^' so that we get predictable versions.

This can be done globally:

```
npm config set save-prefix=''
```

Or we should use .npmrc (I've added that in #36)